semtypes/src/test/java/org/ballerina/semtypes/SemtypeTests.java

```diff
@@ -0,0 +1,18 @@
+﻿package org.ballerina.semtypes;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Tests if placeholder is created.
+ *
+ */
+public class SemtypeTests {
+
+    @Test
+    public void testPlaceholder() {
+        SemtypePlaceholder sp = new SemtypePlaceholder();
```

Comment:
I think we'll want to directly instantieate UniformTypeBitSet , not the place holder